### PR TITLE
add missing trailing slash

### PR DESCRIPTION
**Pull request by ascott1**
Adds a missing trailing slash in the Quick Start section and removes some trailing whitespace

### CODE REVIEW

**Comment by cmc333333** [comment]
Probably a good time to s/127.0.0.0/127.0.0.1 as well

**Comment by ascott1** [comment]
Good catch @cmc333333 . Added.

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ tweaked to pass the parser.
 1. `cd regulations-parser`
 1. `git clone https://github.com/cfpb/fr-notices.git`
 1. `pip install -r requirements.txt`
-1. `echo "LOCAL_XML_PATHS = ['fr-notices']" >> local_settings.py`
+1. `echo "LOCAL_XML_PATHS = ['fr-notices/']" >> local_settings.py`
```
Comment (khandelwal) [inline README.md]: Does it make sense to add a note here about how that trailing slash is required? 
Also, might be useful to put in an issue somewhere about removing the need for the trailing slash.

Comment (ascott1) [inline README.md]: It probably does, but not in the quick start section since it is basically a list of copy and paste-able instructions.

Comment (ascott1) [inline README.md]: Issue created #273

**Comment by willbarton** [comment]
Looks good to me :+1:

**Comment by ascott1** [comment]
I vote for merging before we do anything about #273.

**Pull request merged**